src/components/molecule/CardComponent/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ImageComponent from '../../atom/ImageComponent';
+
+const defaultBorderRadius = "50%";
```

Comment:
Is there a reason why you decided to set this value here as against using styled-components?

Reply:
I was trying to set some sort of default value for the card component but looks like that is not needed. Let's remove that for now.